packages/compass-home/src/components/home.tsx

```diff
@@ -169,7 +166,7 @@ function Home({
   appName: string;
   getAutoConnectInfo?: () => Promise<ConnectionInfo | undefined>;
 }): React.ReactElement | null {
-  const appRegistry = useAppRegistryContext();
+  const appRegistry = useLocalAppRegistry();
```

Comment:
While super not obvious at a glance, home specifically expects a global app registry here, and even though it does work with the top level registry provider either way (on purpose), we might want to still `useGlobalAppRegistry` here for consistency 🤔 This will also have a benefit of not requiring you to wrap every test in a provider

Reply:
So … if I go that route, that works, but then the tests needs adjusting because right now they’re expecting the event listeners to be added to the appRegistry that was explicitly passed in (to `AppRegistryContext.Provider` before this patch), which iiuc would be the `localAppRegistry` for this component, right?

Would you prefer to still make that change? One thing that makes this seem a bit less than ideal here is that the `globalAppRegistry` is a true singleton and doesn’t reset between tests.

Reply:
Ah, that's a good point, thanks for the context! Yeah, from that perspective keeping it with local makes more sense and makes testing easier